README.md

```diff
@@ -1 +1,5 @@
-WIP
```

Comment:
We need to keep the "WIP" comment until the first alpha release.

Reply:
Done ! ;)